testlib/infra.py

```diff
@@ -169,6 +169,36 @@ def setUp(self):
         exec_command(["udevadm", "settle"])
 
 
+class SysfsMonitor(unittest.TestCase):
+    """
+    Manage verification of sysfs files for devices.
+    """
+
+    def tearDown(self):
+        if SysfsMonitor.verify_sysfs:  # pylint: disable=no-member
+            dm_devices = {}
+            for dmdev in os.listdir("/dev/mapper"):
+                dmdev_path = os.path.join("/dev/mapper", dmdev)
+                dmdev_num = os.path.basename(os.path.realpath(dmdev_path))
+                dm_devices[dmdev_num] = dmdev
+
+            try:
+                misaligned_devices = []
+                for dev in os.listdir("/sys/class/block"):
+                    if fnmatch.fnmatch(dev, "dm-*"):
+                        dev_sysfspath = os.path.join(
+                            "/sys/class/block", dev, "alignment_offset"
+                        )
+                        with open(dev_sysfspath, "r", encoding="utf-8") as dev_sysfs:
+                            dev_align = dev_sysfs.read().rstrip()
+                            if int(dev_align) != 0:
+                                misaligned_devices.append(dm_devices[dev])
```

Comment:
Try making the elements of the misaligned devices list a formatted string for now, like: ```f"Stratis name: {dm_devices[dev]}, DM name: ..."``` and possibly the non-zero alignment as well, whatever it is.

Reply:
Resolved.